extra-utils/autojump-rs/autobuild/beyond

```diff
@@ -0,0 +1,29 @@
+abinfo "Extracting assets ..."
+mkdir -pv "$SRCDIR"/../assets/
+tar xvf "$SRCDIR"/../assets.tar.gz --directory "$SRCDIR"/../assets
+
+abinfo "Installing assets ..."
+mkdir -pv "$PKGDIR"/usr/share/autojump
+for shell in bash zsh fish; do
+    install -Dvm644 "$SRCDIR"/../assets/autojump-release-*/bin/autojump.$shell "$PKGDIR"/usr/share/autojump/
+done
+
+abinfo "Installing assets for bash ..."
+mkdir -pv "$PKGDIR"/etc/bashrc.d/
+pushd "$PKGDIR"/etc/bashrc.d/
+ln -sv ../../usr/share/autojump/autojump.bash
+popd
+
+abinfo "Installing assets for fish ..."
+mkdir -pv "$PKGDIR"/usr/share/fish/completions
+mkdir -pv "$PKGDIR"/etc/fish/conf.d
+pushd "$PKGDIR"/usr/share/fish/completions/
+ln -sv ../../autojump/autojump.fish
+popd
+pushd "$PKGDIR"/etc/fish/conf.d
+ln -sv ../../../usr/autojump/autojump.fish
+popd
+
+abinfo "Installing assets for zsh ..."
+mkdir -pv "$PKGDIR"/usr/share/zsh/site-functions
+install -Dvm644 "$SRCDIR"/../assets/autojump-release-*/bin/_j "$PKGDIR"/usr/share/zsh/site-functions/
```

Comment:
That file name could be problematic. @liushuyu your insight would be much appreciated.